tests/test_teacher.py

```diff
@@ -48,3 +48,45 @@ def test_GroundAtomsHold():
     assert len(response.holds) == 2
     assert response.holds[is_block_block]
     assert not response.holds[covers_block_target]
+
+
+def test_DemonstrationQuery():
+    """Tests for answering queries of type DemonstrationQuery."""
+    utils.reset_config({"env": "cover", "approach": "unittest"})
+    teacher = Teacher()
+    env = create_env("cover")
+    task = env.get_train_tasks()[0]
+    state = task.init
+    goal = task.goal
+    # Test normal usage
+    query = DemonstrationQuery(goal)
+    response = teacher.answer_query(state, query)
+    assert isinstance(response, DemonstrationResponse)
+    assert response.query is query
+    assert isinstance(response.teacher_traj, LowLevelTrajectory)
+    print(len(response.teacher_traj.actions))
+    assert len(response.teacher_traj.actions) == 2
+    # (leverage GroundAtomsHoldQuery to test if the teacher_traj
+    # successfully achieves the goal)
+    goal_query = GroundAtomsHoldQuery(goal)
+    goal_holds_response = teacher.answer_query(
+        response.teacher_traj.states[-1], goal_query)
+    assert False not in goal_holds_response.holds.items()
```

Comment:
I don't think False can ever be in the items(). do you mean values() instead? better to write this in a more explicit way: `assert goal_holds_response.holds == {goal_query: False}` or something

Reply:
changed to just use the convenience `holds` function with the ground atoms in the goal